website/docs/cloud-docs/registry/publish-providers.mdx

```diff
@@ -20,72 +20,196 @@ Users must be members of an organization to access its registry and private prov
-      https://archivist.terraform.io/v1/object/dmF1b64hd73ghd63
-    ```
-
-1. Use the [Create a Provider Platform endpoint](/terraform/cloud-docs/api-docs/private-registry/provider-versions-platforms#create-a-provider-platform) to create a platform for the version. Platforms are binaries that allow the provider to run on a particular operating system and architecture combination (e.g., Linux and AMD64). The response includes a `provider-binary-upload` URL that you will use to upload the binary file for the platform.
-
-    ```json
-    "links": {
-          "provider-binary-upload": "https://archivist.terraform.io/v1/object/dmF1b45c367djh45nj78"
-        }
-    ```
-
-1. Upload the platform binary file to the URL from the previous step. The example command below uploads the binary from a local machine.
-
-    ```
-    $ curl -T local-example/terraform-provider-random_3.1.3_linux_amd64.zip
-      https://archivist.terraform.io/v1/object/dmF1b45c367djh45nj78
-    ```
-
-1. (Optional) Repeat steps 4 and 5 to add additional platform binaries for the release.
+Before consumers can use a private provider, you must create the provider, upload a GPG signing key, create at least one version, create at least one platform for that version, and upload release files.
 
-The version is available in the Terraform Cloud UI. Consumers can begin using this provider version in configurations.
+### Create the provider
+
+Create a file named `provider.json` with the following contents. Replace `PROVIDER_NAME` with the name of your provider and replace `ORG_NAME` with the name of your organization.
+
+```json
+{
+  "data": {
+    "type": "registry-providers",
+    "attributes": {
+      "name": "PROVIDER_NAME",
+      "namespace": "ORG_NAME",
+      "registry-name": "private"
+    }
+  }
+}
+```
+
+Use the [Create a Provider endpoint](/terraform/cloud-docs/api-docs/private-registry/providers#create-a-provider) to create the provider in Terraform Cloud. Replace `TOKEN` in the `Authorization` header with your Terraform Cloud API token and replace `ORG_NAME` with the name of your organization.
+
+```shell-session
+$ curl \
+  --header "Authorization: Bearer TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @provider.json \
+  https://app.terraform.io/api/v2/organizations/ORG_NAME/registry-providers
+```
+
+The provider is now available in your organization’s Terraform Cloud private registry, but consumers cannot use it until you add a version and a platform. To create them, you will will need the provider binaries, public GPG signing key, `SHA256SUMS` file, and `SHA256SUMS.sig` file from at least one release.
+
+
+### Add your public key
+
+-> **NOTE**: If you are uploading a public provider into your private registry, skip this step and instead use [HashiCorp's public PGP key](https://www.hashicorp.com/.well-known/pgp-key.txt). To retrieve the `key-id` of HashiCorp's public PGP key, TODO
```

Comment:
Let's add a link to the next step to this note. Something to make it stand out a little bit more. I skimmed right past it the first time I read the page.

It may also not be necessary to hyperlink to the key itself, since we include it for you. The important thing for this note to convey is, skip this step, use this key-id in the next step.